### DISCUSSION TITLE
画布中使用了disableSelection()，但是只是高亮的Selection不可用，节点仍然可以选中后变形

### DISCUSSION THREAD

**Discussion by ohha36** [antvis/X6#3414] [Q&A]
使用了disableSelection()后，只是高亮的选取框没有，并且节点不可移动，但是选择后，节点仍然可以变形，如下图。
需要使用什么方法使该节点完全不可操作？谢谢。
![image](https://user-images.githubusercontent.com/49863030/226852205-38061648-d87f-40eb-8d71-9b234e02f57c.png)

**Comment by NewByVector** [comment]
如果是 2.x 版本，不要使用  x6-plugin-transform 插件，如果是 1.x 版本，关闭 resizing 配置。

**Reply by ohha36** [reply]
解决了，谢谢~！
